README.md

```diff
@@ -13,40 +15,59 @@ A [NodeCG](http://github.com/nodecg/nodecg) bundle.
 
 ## Install
 
-1.1. Install NodeCG and [nodecg-cli](https://github.com/nodecg/nodecg-cli) (optional)
+- Install [git](http://git-scm.com/)
```

Comment:
It may be best for this, that we update the overlay control readme first and do the update on all the overlays reliant on 2.0 at once to keep it consistent